app/components/shared/Slider.vue.ts

```diff
@@ -22,8 +24,12 @@ export default class SliderInput extends Vue {
   mounted() {
     // Hack to prevent transitions from messing up slider width
     setTimeout(() => {
-      if (this.$refs.slider) this.$refs.slider.refresh();
+      this.onResizeHandler();
     }, 500);
+
+    new ResizeSensor(this.$el, () => {
+      this.onResizeHandler();
+    });
```

Comment:
instead of making an anonymous function you can just do `ResizeSensor(this.$el, this.onResizeHandler);`

also is there a reason to use this library vs `document.addEventListener('resize', this.onResizeHandler)`?

Reply:
the `resize` event actually is window-resize event. When chat panel is expanding/collapsing the window size is not changing.